tensorflow/java/src/main/java/org/tensorflow/SavedModelBundle.java

```diff
@@ -36,9 +36,22 @@
    * @return a bundle containing the graph and associated session.
    */
   public static SavedModelBundle load(String exportDir, String... tags) {
-    return load(exportDir, tags, null);
+    return load(exportDir, tags, null, null);
   }
-
+   
+    /**
+   * Load a saved model from an export directory. The model that is being loaded should be created
+   * using the <a href="https://www.tensorflow.org/api_docs/python/tf/saved_model">Saved Model
+   * API</a>.
+   *
+   * @param exportDir the directory path containing a saved model.
+   * @param tags the tags identifying the specific metagraphdef to load.
+   * @return a bundle containing the graph and associated session.
+   */
+  public static SavedModelBundle load(String exportDir,byte[] runOptions, byte[] sessionOptions, String... tags) {
```

Comment:
Mind formatting the file as per [`google-java-format`](https://github.com/google/google-java-format)?